test/unit/gateways/paysafe_test.rb

```diff
@@ -121,6 +131,16 @@ def test_successful_verify
     assert_equal '493936', response.params['authCode']
   end
 
+  def test_successful_store
+    @gateway.expects(:ssl_request).returns(successful_store_response)
+
+    response = @gateway.store(@credit_card, @more_options)
+    assert_success response
+
+    assert_equal '111-222-3456', response.params['phone']
```

Comment:
This test isn't really covering your code, since you are providing the `successful_store_response` when the request is made. If you remove `phone` from `@more_options` it still passes. I'm not sure if that was the intention behind this test, so my apologies if I am misreading it.

You could do a `check_request`-style test to verify that a `phone` being passed in at the `options` or `options[:billing_address]` level makes it into the post.